azure-pipelines.bundlesize.yml

```diff
@@ -8,6 +8,36 @@ variables:
   - template: .devops/templates/variables.yml
 
 jobs:
+  - job: bundlesize
+    workspace:
+      clean: all
+    pool: 'Self Host Ubuntu'
+    steps:
+      - template: .devops/templates/tools.yml
+
+      - task: Bash@3
+        inputs:
+          filePath: yarn-ci.sh
+        displayName: yarn
+
+      - script: |
+          yarn build --no-cache $(sinceArg)
+        displayName: build packages
+
+      - script: |
+          yarn lage bundle-size --no-cache
+        displayName: create reports for packages
+
+      - script: |
+          yarn bundle-size upload-report --branch=$(Build.SourceBranchName) --commit-sha $(Build.SourceVersion)
+        displayName: upload a report (base only)
+        condition: not(variables.isPR)
+        env:
+          BUNDLESIZE_ACCOUNT_KEY: $(bundlesize-account-key)
```

Comment:
I would add this as a comment: https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#secret-variables

Since if someone not used to ADO is looking, it's not obvious where this comes from

Reply:
Added in 3537d55.